components/meter/meter-styles.js

```diff
@@ -0,0 +1,16 @@
+import { css } from 'lit-element/lit-element.js';
```

Comment:
I'm guessing we're going to use these styles for the other variants as well?

Reply:
Yes. All of them have the same styles.